google-api-client/pom.xml

```diff
@@ -127,10 +128,20 @@
       <artifactId>commons-codec</artifactId>
       <scope>provided</scope>
       <version>1.6</version>
-   </dependency>
+    </dependency>
     <dependency>
       <groupId>com.google.guava</groupId>
       <artifactId>guava-jdk5</artifactId>
     </dependency>
+    <dependency>
+      <groupId>javax.json</groupId>
```

Comment:
We should specifically avoid these javax.json dependencies, since this version of the API must run on a broad set of platforms that can have restrictions about this. Make use of the existing com.google.api.client.json instead.